templates/.gitignore.j2

```diff
@@ -1,4 +1,6 @@
 reports/
+doc/
```

Comment:
In my PR it will be publishing within `docs/` and above in `.npmignore`, it is also as `docs/`, so I think that should be changed here (and in the built `.gitignore`).